cluster.yml

```diff
@@ -27,7 +27,7 @@
   roles:
     - { role: kubespray-defaults}
     - { role: kubernetes/preinstall, tags: preinstall }
-    - { role: docker, tags: docker }
+    - { role: docker, tags: docker, when: "docker_enabled | default(true)" }
```

Comment:
You're setting a class default, so there's no need to set a default here.
Can you change this line to:
```
    - { role: docker, tags: docker, when: docker_enabled } 
```

Reply:
@mattymo, the reason I did it this way is because k8s-cluster.yml is a user-modified file, so I figured it would be a good idea to be tolerant to a missing variable, in case somebody is using an old k8s-cluster.yml.

Your call, though.